README.md

```diff
@@ -348,9 +348,11 @@ The `usemin` task has 2 actions:
 
 ### Finding references
 
-`usemin` will look for different kinds of references in files of certain types. The type of the file should be indicated by using target name. That is to say, the target name in the `usemin` task should only be in `html`, `css`, `json` and `svg` so that `usemin` can find possible types of references in these files.
+`usemin` will look for different kinds of references in files of certain types. The type of the file should be indicated by using target name. That is to say, any target name in the `usemin` task should only be in `html`, `css`, `json` and `svg` so that `usemin` can lookup possible types of references in these files of a certain type.
 
-For example, if the target name is `html`, `usemin` will find references of images, scripts, stylesheets, etc. However, `usemin` will only find references look like `url(...)` when target name is `css`.
+For example, if the target name is `html`, `usemin` will find references of `<img src="..."`, `<script src="..."`, `<link href="..."`, etc.
+However, `usemin` will only find references look like `url(...)` when target name is `css`.
```

Comment:
"However, `usemin` will only find references **which** look like `url(...)` when **the** target name is `css`."

This might clear up some confusion.